transaction.go

```diff
@@ -110,9 +110,14 @@ type transactionCanonicalForm struct {
 	EnvelopeSignatures []transactionSignatureCanonicalForm
 }
 
+// DefaultTransactionGasLimit should be high enough for small transactions
+const DefaultTransactionGasLimit = 1000
```

Comment:
Maybe this should match the behavior of the flow-go, which is to use the default limit? 9999 as far as I remember

Reply:
I didn't want to use the maximum as the default... but I guess right now there is no reason not to?